src/api/forms/repositories/form-metadata-repository.js

```diff
@@ -18,7 +18,14 @@ export function list() {
     db.collection(METADATA_COLLECTION_NAME)
   )
 
-  return coll.find().limit(MAX_RESULTS).toArray()
+  return coll
+    .find()
+    .sort({
+      updatedAt: -1, // Primary sort: most recently updated documents first (-1 = descending order)
+      'updatedBy.displayName': 1 // Secondary sort: if update times are equal, sort by author name (1 = ascending order)
```

Comment:
LGTM - I'm not sure this secondary sort criteria is necessary though?

Reply:
Fair point. It's almost never going to be the case that two forms are updated at the exact same moment.

@mokhld is this meant to be a sort on date + author rather than datetime + author?

Reply:
Yeah exactly @alexluckett. It's almost a rarity if ever as we're not high traffic at all, however it still covers the day/month/year etc.

Reply:
`updatedAt: -1` is indexing a Date+Time field, so wouldn't two forms have to have exactly the same updatedAt value (down to the nanosecond) before that second sort index is needed to discriminate.

Reply:
Sorry should have clarified, the date/time comment was good, was just explaining the secondary sort

@mokhld Do you mind checking if the grouping-by-author is working?

Reply:
@colinrotherham Yep it is with my tests. @davidjamesstone You're spot on btw, the secondary sort will prob never fire tbh and it could well be taken out with these new library refresh stories if needed however is does cover that extremely rare occurrence.